### PR TITLE
Testing front end

### PR DESCRIPTION
**Pull request by Anshiboy**
### Description

The test files for every single screen and component used in the front-end

### Changes Made

Created individual test files for all screens, and components while intrinsically covering dark mode and store. Do not know what to do with app.test.js, but can be deleted later if not needed. Overall 100% coverage for all components, but only 65% coverage for screens as there were a few hiccups with the connection with backend. Also Document viewer has not been tested yet as a PR to implement it into FilesScreen will be made very shortly.

### Screenshots (if applicable)

image

Coverage for all ^

image

Tests passing ^ (the one suite that fails is App.js as there is no tests that can be performed for it yet)
### Checklist

- [x] I have tested the changes locally.
- [x] My code follows the project's coding style.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings or errors.
- [x] I have added tests that prove my fix is effective or that my feature works.

### Additional Notes (if any)

npm run test a -- --coverage

to see coverage and test yourself ^^

### CODE REVIEW

File: package.json
Comment (GregoryGismondi) [inline package.json:1]: Changes look good, no issues or changes with other packages

File: src/App.test.js
Comment (GregoryGismondi) [inline src/App.test.js:1]: File does not have any references so it's deletion is safe

**Review by GregoryGismondi — APPROVED** [review]
# Code Review
## Coding style
Proper coding style and conventions were followed

## Correctness (check the branch and try running the code)
All tests ran and returned a success except App.test.js. However, there are no tests inside to be run which is why the program fails. We can delete src/\_\_test\_\_/App.test.js in a future commit. Below is a screenshot of me running the test locally:

image

Changes to the package.json does not affect other parts of the program. 

The deletion of App.test.js in src folder is safe as it has no references and it was not completed

Update: src/\_\_test\_\_/App.test.js has been removed with no issues. Below are the tests ran again (now without src/\_\_test\_\_/App.test.js): 

image

## Documentation
Documentation for changes and the result of test cases are shown properly with a screenshot

## Clarity of pull request
All commits of the pull request are shown and all code is commented. Nice job overall!

**Comment by GregoryGismondi** [comment]
Updated approval comment to fit updated commit

**Review by sarvaSanjay — CHANGES_REQUESTED** [review]
Keep this on hold till further changes are merged. MVP is priority and I don't want the team working on that to have merge conflicts/ other issues

**Comment by GregoryGismondi** [comment]
> Keep this on hold till further changes are merged. MVP is priority and I don't want the team working on that to have merge conflicts/ other issues

Sounds good! Will re-review any changes that need to be made once MVP is complete

**Comment by Anshiboy** [comment]
> Keep this on hold till further changes are merged. MVP is priority and I don't want the team working on that to have merge conflicts/ other issues

fixed

**Review by sarvaSanjay — APPROVED** [review]
As MVP is complete, we can merge. Good job connecting all changes

**Pull request merged**